sklearn/preprocessing/data.py

```diff
@@ -350,6 +466,8 @@ class StandardScaler(BaseEstimator, TransformerMixin):
     that others, it might dominate the objective function and make the
     estimator unable to learn from other features correctly as expected.
 
+    This scaler can also be applied to sparse CSR or CSC matrices.
```

Comment:
by passing `with_mean=False` to avoid breaking the sparsity structure of the data.